source/init/mus_initLBMPS_module.f90

```diff
@@ -47,10 +49,11 @@ module mus_initLBMPS_module
 
 ! ****************************************************************************** !
   !> Initialize the relaxation model for lbm passive scalar scheme kind
-  subroutine mus_init_advRel_lbm_ps( relaxation, layout, compute )
+  subroutine mus_init_advRel_lbm_ps( relaxation, layout, order, compute )
     ! ---------------------------------------------------------------------------
     character(len=labelLen), intent(in) :: relaxation
     character(len=labelLen), intent(in) :: layout
+    character(len=labelLen), intent(in) :: order
```

Comment:
same as previous comment. To be replaced with 
`character(len=labelLen), intent(in) :: :: relaxation_variant`